src/main/java/org/folio/consortia/service/KeycloakUsersService.java

```diff
@@ -0,0 +1,13 @@
+package org.folio.consortia.service;
+
+public interface KeycloakUsersService {
+
+  /**
+   * Migrate users to create identity provider links
+   *
+   * @param tenantId tenant id to migrate users from
+   * @param centralTenantId central tenant id to migrate users to
+   */
+  void migrateUsers(String tenantId, String centralTenantId);
```

Comment:
currently we do not migrate users, just create identity links for them, so can rename to createUsersIdpLinks